src/pybind/mgr/dashboard/frontend/src/app/shared/services/prometheus-alert-formatter.ts

```diff
@@ -65,7 +65,8 @@ export class PrometheusAlertFormatter {
       info: ['suppressed', 'unprocessed'],
       success: ['resolved']
     };
-    return NotificationType[_.findKey(types, (type) => type.includes(status))];
+    const key = _.findKey(types, (type) => type.includes(status));
+    return _.get(NotificationType, key);
```

Comment:
I wonder why this change is needed, as I don't see the difference and it's not more type save than before. The only difference is that `_.get` is taking a view more steps.

Reply:
With `noImplicitAny `enabled and `suppressImplicitAnyIndexErrors` disabled, TS is very strict with the type of the index used.
p.e. in a normal class you can't use any index to access its properties.

Since this is no longer the case I will revert.